app/liquid/liquid_renderer.rb

```diff
@@ -19,22 +19,27 @@ def template
     @template ||= Liquid::Template.parse(@layout.content)
   end
 
+  def markup_data
+    return @markup_data if @marup_data
+
+    @markup_data = Rails.cache.fetch( cache.key_for_data ) do
+      plugin_data = Plugins.data_for_view(@page, {form_values: @member.try(:attributes), donation_band: @url_params[:donation_band]})
```

Comment:
this line doesn't cache. notice how we're passing the attributes of the member - those get passed to the plugins. If a plugin has a form, it runs the member attributes against the validator for the form to determine which fields it can skip.